package.json

```diff
@@ -80,8 +50,10 @@
     "vega-tooltip": "^0.9.16"
   },
   "scripts": {
-    "start": "node scripts/start.js",
-    "build": "node scripts/build.js",
+    "start": "webpack-dev-server --mode development --hot --inline --progress --open --color",
+    "build": "webpack --mode production --progress",
+    "clean": "rm -rf build",
+    "postbuild": "cp -rn public/* build",
```

Comment:
Why do we need a build folder and don't build directly into the public folder?

Reply:
Ahh, I see. Maybe better name it `deploy`.

Reply:
`/dist: "distribution", the compiled code/library`
I think `dist` is like more standard.